<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>casedialog</class>
 <widget class="QWidget" name="casedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1419</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelDate">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoctor">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDep">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditResult">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>1211</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="insturctions">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>50</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>查看医嘱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prescription">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>240</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>查看处方</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
